input/docs/extending/modules/best-practices.md

```diff
@@ -102,12 +102,11 @@ The wording of each guideline indicates how strong the recommendation is:
 
 ## .NET target version
 
-**_§2.5_** **Do** target `netstandard2.0`.
+**_§2.5_** **Do** target `netcoreapp3.1`.
```

Comment:
Is targeting only `netcoreapp3.1` what we suggest due to the issues with multi-targeting?

Reply:
Yes, that seems to be the right choice until we drop support for `netcoreapp3.1`.